checker/tests/nullness-enclosingexpr/NullnessEnclosingExprTest.java

```diff
@@ -0,0 +1,39 @@
+import org.checkerframework.checker.initialization.qual.Initialized;
+import org.checkerframework.checker.initialization.qual.UnknownInitialization;
+
+class NullnessEnclosingExprTest {
+    class InnerWithImplicitEnclosingExpression {
+        // This will lead to a NPE, since NullnessEnclosingExprTest
```

Comment:
This should never lead to an NPE, because the enclosing expression is `@Initialized`. It was leading to an NPE, because the instantiation with an un-initialized enclosing expression was allowed.
So the correct comment here is that there is no possible NPE and therefore no expected error.

Reply:
Thanks!